ui/public/scripts/components/mobile-app/enable-mobile-build.component.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+/**
+ * @ngdoc component
+ * @name mcp.component:enable-mobile-build
+ * @description
+ * # enable-mobile-build
+ */
+angular.module('mobileControlPanelApp').component('enableMobileBuild', {
+  template: `<div class="blank-slate-pf" id="">
+              <div class="blank-slate-pf-icon">
+                <span class="pficon pficon pficon-add-circle-o"></span>
+              </div>
+              <h1>
+                Enable Mobile CI/CD Service
+              </h1>
+              <p>
+                To enable mobile application builds, please provision the Mobile CI/CD service.
+              </p>
+              <p>
+                This can be provisioned via the application catalog.
+              </p>
+              <p>
+                Learn more about this <a href="http://feedhenry.org/docs/">in the documentation</a>.
+              </p>
+              <div class="blank-slate-pf-main-action">
+                <a ng-href="/" class="btn btn-primary btn-lg">Application Catalog</a>
```

Comment:
This button might read better as 'Provision Mobile CI/CD Service'

Reply:
Updated